docker/Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM swift:5.1.3 as builder
```

Comment:
Maybe we should update this one? `5.1.5`?

Reply:
Is 5.1.5 out? I thought we were at 5.1.4

Reply:
5.1.5 is out:
https://swift.org/download/

Reply:
I set 5.1.5 on master branch. Didn't want to hide change in another PR